unirest.gemspec

```diff
@@ -1,22 +1,22 @@
 Gem::Specification.new do |s|
   s.name        = 'moesif_unirest'
-  s.version     = '1.1.5'
+  s.version     = '1.1.6'
   s.summary     = "Unirest-Ruby"
   s.description = "Simplified, lightweight HTTP client library"
   s.authors     = ["Moesif, Inc"]
   s.email       = 'derric@moesif.com'
   s.homepage    = 'https://github.com/Moesif/unirest-ruby'
   s.license     = 'MIT'
 
-	s.add_dependency('rest-client', '>= 1.8.0')
+  s.add_dependency('rest-client', '>= 1.8.0')
   s.add_dependency('json', '>= 1.8.1')
-	s.add_dependency('addressable', '>= 2.3.5')
+  s.add_dependency('addressable', '~> 2.8.0')
 
-	s.add_development_dependency('shoulda', '~> 3.5.0')
-	s.add_development_dependency('test-unit')
-	s.add_development_dependency('rake')
+  s.add_development_dependency('shoulda', '~> 3.5.0')
+  s.add_development_dependency('test-unit')
```

Comment:
In this PR https://github.com/Moesif/moesifapi-ruby/pull/13/files `test-unit` has version dependency but not here. Let's have a consistency and add version here too.